gdal/frmts/jpegxr/jpegxrdataset.cpp

```diff
@@ -215,12 +215,23 @@ struct JPEGXRDecoder final
         if (nErr != WMP_errSuccess || !pImageDecode)
             throw JPEGXRException(nErr);
 
+        PKPixelInfo pi;
+        GetPixelInfo(pi);
+
+        // Alpha
+        // 0: Decode without alpha channel
+        // 1: Decode only alpha channel (TODO: unused, might be controlled with driver option)
+        // 2: Decode image & alpha (default)
+        if (!!(pi.grBit & PK_pixfmtHasAlpha))
```

Comment:
Mostly a matter of taste, but I feel that x != 0 is more readable than !!x (and more consistant with what can be found in other parts of the code base)